bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/sensors/DimmerSwitchHandler.java

```diff
@@ -96,7 +77,7 @@ protected void doSensorStateChanged(@Nullable HueBridge bridge, FullSensor senso
             String value = String.valueOf(buttonState);
             updateState(CHANNEL_DIMMER_SWITCH, new DecimalType(value));
             Instant then = timestamp.toInstant();
-            Instant someSecondsEarlier = now.minusNanos(refreshIntervalInNanos).toInstant();
+            Instant someSecondsEarlier = now.minusSeconds(5).toInstant();
```

Comment:
5 seems like a random number - if I read the old code correctly, the intention was to use half of the refresh interval.

If you think 5 is always a good choice, you'd better define it as a constant with some comment next to it.

Reply:
I added a comment inside the code.